src/api/svg.ts

```diff
@@ -1570,6 +1564,10 @@ const parseHTMLNode = (
       ...attributes.inherited,
       ...attributes.svgAttributes,
     };
+    if (node.tagName === 'polygon') {
+      svgAttributes.d = `M${svgAttributes.points?.replace(/ /g, 'L')}Z`;
+      node.tagName = 'path';
+    }
```

Comment:
It's better to place this code before the attributes parsing (line 1568). This way, we can remove the parsing of the attribute `points` (lines 1148 - 1165), which is duplicating the code that parses the `d` attribute.

The code for converting should be:
```js
if (node.tagName === 'polygon') {
    node.attributes.d = `M${node.attributes.points}Z`; 
    delete node.attributes.points
    node.tagName = 'path';
}
```
I think you don't need to convert empty spaces to `L`.

About the issue related to the cropping of a path, I understand the issue. It's just a small detail that should be fixed on the cropping algorithm. It's not related to your changes.